### PR TITLE
Feature/add id element

### PR DESCRIPTION
**Pull request by gazizova-vlada**
- Added missing id/className for autotests
- Rename settings "common" to "personal", "admin" to "general"

### CODE REVIEW

File: packages/client/src/components/dialogs/SalesDepartmentRequestDialog/index.js [hunk 1]
```diff
@@ -110,6 +110,7 @@ const SalesDepartmentRequestDialog = ({
           errorMessage={t("Common:RequiredField")}
         >
           <TextInput
+            id="name"
```
Comment (AlexeySafronov) [inline packages/client/src/components/dialogs/SalesDepartmentRequestDialog/index.js:113]: Too simple id value (potential duplicates)

File: packages/client/src/pages/PortalSettings/categories/data-management/backup/common-container/DirectThirdPartyConnection.js [hunk 2]
```diff
@@ -364,6 +364,7 @@ const DirectThirdPartyConnection = (props) => {
 
       {!connectedThirdPartyAccount?.id || !isTheSameThirdPartyAccount ? (
         <Button
+          id="connect"
```
Comment (AlexeySafronov) [inline packages/client/src/pages/PortalSettings/categories/data-management/backup/common-container/DirectThirdPartyConnection.js:367]: Too simple id value (potential duplicates)

File: packages/client/src/pages/PortalSettings/categories/data-management/backup/manual-backup/index.js [hunk 3]
```diff
@@ -298,6 +299,7 @@ class ManualBackup extends React.Component {
           {isCheckedTemporaryStorage && (
             <div className="manual-backup_buttons">
               <Button
+                id="create"
```
Comment (AlexeySafronov) [inline packages/client/src/pages/PortalSettings/categories/data-management/backup/manual-backup/index.js:302]: Too simple id value (potential duplicates)

File: packages/client/src/pages/PortalSettings/categories/integration/SingleSignOn/sub-components/AddIdpCertificateModal.js [hunk 4]
```diff
@@ -36,6 +36,7 @@ const AddIdpCertificateModal = (props) => {
 
         <TextArea
           className="text-area"
+          id="idpCertificate"
```
Comment (AlexeySafronov) [inline packages/client/src/pages/PortalSettings/categories/integration/SingleSignOn/sub-components/AddIdpCertificateModal.js:39]: Rename "idpCertificate" to "idp-certificate"

File: packages/client/src/pages/PortalSettings/categories/integration/SingleSignOn/sub-components/AddSpCertificateModal.js [hunk 5]
```diff
@@ -51,6 +51,7 @@ const AddSpCertificateModal = (props) => {
         </Text>
 
         <TextArea
+          id="spCertificate"
```
Comment (AlexeySafronov) [inline packages/client/src/pages/PortalSettings/categories/integration/SingleSignOn/sub-components/AddSpCertificateModal.js:54]: Rename "spCertificate" to "sp-certificate"

File: packages/client/src/pages/PortalSettings/categories/integration/SingleSignOn/sub-components/AddSpCertificateModal.js [hunk 6]
```diff
@@ -64,6 +65,7 @@ const AddSpCertificateModal = (props) => {
         </Text>
 
         <TextArea
+          id="spPrivateKey"
```
Comment (AlexeySafronov) [inline packages/client/src/pages/PortalSettings/categories/integration/SingleSignOn/sub-components/AddSpCertificateModal.js:68]: Rename "spPrivateKey" to "sp-privateKey"

File: packages/client/src/pages/PortalSettings/categories/integration/ThirdPartyServicesSettings/sub-components/consumerModalDialog.js [hunk 7]
```diff
@@ -237,6 +240,7 @@ class ConsumerModalDialog extends React.Component {
           <Button
             primary
             size="normal"
+            id="enable"
```
Comment (AlexeySafronov) [inline packages/client/src/pages/PortalSettings/categories/integration/ThirdPartyServicesSettings/sub-components/consumerModalDialog.js:243]: Too simple id value (potential duplicates)

File: packages/client/src/pages/PortalSettings/categories/integration/ThirdPartyServicesSettings/sub-components/consumerModalDialog.js [hunk 8]
```diff
@@ -246,6 +250,7 @@ class ConsumerModalDialog extends React.Component {
           <Button
             size="normal"
             scale
+            id="cancel"
```
Comment (AlexeySafronov) [inline packages/client/src/pages/PortalSettings/categories/integration/ThirdPartyServicesSettings/sub-components/consumerModalDialog.js:253]: Too simple id value (potential duplicates)

File: packages/client/src/components/dialogs/SalesDepartmentRequestDialog/index.js (hunk 1 above)
Comment (gazizova-vlada) [reply packages/client/src/components/dialogs/SalesDepartmentRequestDialog/index.js]: Fix to "your-name"

File: packages/client/src/pages/PortalSettings/categories/data-management/backup/common-container/DirectThirdPartyConnection.js (hunk 2 above)
Comment (gazizova-vlada) [reply packages/client/src/pages/PortalSettings/categories/data-management/backup/common-container/DirectThirdPartyConnection.js]: Fix to "connect-button"

File: packages/client/src/pages/PortalSettings/categories/data-management/backup/manual-backup/index.js (hunk 3 above)
Comment (gazizova-vlada) [reply packages/client/src/pages/PortalSettings/categories/data-management/backup/manual-backup/index.js:302]: Fix to "create-button"

File: packages/client/src/pages/PortalSettings/categories/integration/SingleSignOn/sub-components/AddIdpCertificateModal.js (hunk 4 above)
Comment (gazizova-vlada) [reply packages/client/src/pages/PortalSettings/categories/integration/SingleSignOn/sub-components/AddIdpCertificateModal.js]: Fiix to "idp-certificate"

File: packages/client/src/pages/PortalSettings/categories/integration/SingleSignOn/sub-components/AddSpCertificateModal.js (hunk 6 above)
Comment (gazizova-vlada) [reply packages/client/src/pages/PortalSettings/categories/integration/SingleSignOn/sub-components/AddSpCertificateModal.js]: Fix to "sp-privateKey"

File: packages/client/src/pages/PortalSettings/categories/integration/SingleSignOn/sub-components/AddSpCertificateModal.js (hunk 5 above)
Comment (gazizova-vlada) [reply packages/client/src/pages/PortalSettings/categories/integration/SingleSignOn/sub-components/AddSpCertificateModal.js]: Fix to "sp-certificate"

File: packages/client/src/pages/PortalSettings/categories/integration/ThirdPartyServicesSettings/sub-components/consumerModalDialog.js (hunk 7 above)
Comment (gazizova-vlada) [reply packages/client/src/pages/PortalSettings/categories/integration/ThirdPartyServicesSettings/sub-components/consumerModalDialog.js]: Fix to "enable-button"

File: packages/client/src/pages/PortalSettings/categories/integration/ThirdPartyServicesSettings/sub-components/consumerModalDialog.js (hunk 8 above)
Comment (gazizova-vlada) [reply packages/client/src/pages/PortalSettings/categories/integration/ThirdPartyServicesSettings/sub-components/consumerModalDialog.js]: Fix to "cancel-button"

**Review by AlexeySafronov — APPROVED** [review]

**Pull request merged**